src/main/java/com/plaid/client/request/InstitutionsGetByIdRequest.java

```diff
@@ -11,10 +11,26 @@
  */
 public final class InstitutionsGetByIdRequest extends BasePublicRequest {
   private String institutionId;
+  private Options options;
 
   public InstitutionsGetByIdRequest(String institutionId) {
     notNull(institutionId, "institutionId");
 
     this.institutionId = institutionId;
   }
+
+  public InstitutionsGetByIdRequest(String institutionId, boolean includeInstitutionData) {
```

Comment:
can we add a .withIncludeInstitutionData(boolean) instead of creating more constructors?

Reply:
yes will do